packages/simple-schema/lib/validators/unknown-attributes.js

```diff
@@ -20,7 +21,10 @@ function validateUnknownAttributes(elem, schema) {
      * reproduce this branch with current error recovery heuristics
      */
     if (attrib.key !== null) {
-      if (includes(allowedAttribNames, attrib.key) === false) {
+      if (
+        includes(allowedAttribNames, attrib.key) === false &&
+        !NAMESPACE_ATTRRIBUTE_PATTERN.test(attrib.key)

```

Comment:
the source code uses `=== false` pattern to:
- avoid boolean conversions
- make it more clear we are checking for **NOT** because the '!' operator in JavaScript is very "thin" and small.